core/server/data/meta/excerpt.js

```diff
@@ -1,19 +1,60 @@
-var downsize = require('downsize');
+/* jshint plusplus: true */
+// something useful: https://babeljs.io/repl/#?evaluate=true&presets=es2015
```

Comment:
This and the comment beneath it can be removed. Especially because the Ghost server instance isn't using es6 😉